configuration/openmrs/i18n/clinical/locale_fr.json

```diff
@@ -388,7 +388,7 @@
   "VISIT_TITLE_TREATMENTS_KEY": "Médicaments",
   "YEARS_LABEL": "An(s)",
   "VACCINATIONS": "Vaccinations",
-  "MOST_RECENT_VACCINATION_SEQUENCES":"SÉQUENCES DE VACCINATION LES PLUS RÉCENTES",
+  "MOST_RECENT_VACCINATION_SEQUENCES":"Séquence de vaccination les plus récentes",
```

Comment:
"Séquence de vaccination les plus récentes" should be "Séquences de vaccination les plus récentes".
Now I am thinking maybe we could take the liberty to name it simply "Most recent vaccinations". Even though this is not acurate, as this only show the most recent sequences of vaccinations, providers probably won't care.
In that case, the french will be "Vaccination récentes", english: "Recent vaccinations"

Reply:
Done